elkodon_pal/posix/src/windows/constants.rs

```diff
@@ -6,6 +6,7 @@ use crate::posix::types::*;
 pub const CPU_SETSIZE: usize = core::mem::size_of::<usize>() * 8;
 pub const MAX_NUMBER_OF_THREADS: usize = 1024;
 pub const FD_SETSIZE: usize = windows_sys::Win32::Networking::WinSock::FD_SETSIZE as _;
+pub const NULL_TERMINATOR: char = 0;
```

Comment:
The `char` is not the Rust primitive `char`, right? I think this might become misleading since Rust developer expect the `char` type to be a unicode scalar value. I'm quite surprised that it is even possible to shadow the primitive types. Surprised is not the right word, I'm kind of shocked that this works
```rust
type i16 = i8;
```

Reply:
Every platform here has its own char defined in `elkodon_pal/posix/src/{OS}/types.rs` I would leave it for now and refactor it later.